hooks/024-configure-bootchart.chroot

```diff
@@ -0,0 +1,75 @@
+#! /bin/sh -ex
+
+# Rewrite configuration and unit that came with the systemd-bootchart package
+
+cat << 'EOF' > /etc/systemd/bootchart.conf
+[Bootchart]
```

Comment:
I have changed output to `/run/log/base` now. Saving directly to `ubuntu-save` is a bit more tricky because maybe the mountpoint is not available when `ExecStartPre` from `systemd-bootchart.service` is run, which is the point where the output folder would be created.

The nice thing with the change is that now we do not need systemd-bootchart-prestart.sh :)